### PR TITLE
fix: Increased contrast for transport colors for Troms

### PR DESCRIPTION
**Pull request by tft-abel**
Swaps Region (now green) and City (now blue) colors and for both uses a shade that has white text on them. I updated this in Figma for Troms' design system too

### CODE REVIEW

File: packages/assets/package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@atb-as/theme": "^10.3.2",
+    "@atb-as/theme": "^10.3.3",
```
Comment (mortennordseth) [inline packages/assets/package.json:30]: Did you bump the versions manually? This is done automatically when publishing the new version with `yarn pub` after this has been merged into the main branch :)

File: packages/theme/src/themes/troms-theme/theme.ts [hunk 2]
```diff
@@ -191,13 +191,13 @@ const themes: Themes = {
...
-        secondary: baseColors.green_800,
-      },
       transport_region: {
+        primary: baseColors.green_800,
+        secondary: baseColors.green_900,
+      },
+      transport_city: {
         primary: baseColors.blue_400,
-        secondary: baseColors.blue_400,
+        secondary: baseColors.blue_500,
```
Comment (mortennordseth) [inline packages/theme/src/themes/troms-theme/theme.ts:200]: You should run `yarn build-all` to apply these changes to the .css-files and include the changes to this PR 👍

File: packages/assets/package.json (hunk 1 above)
Comment (tft-abel) [reply packages/assets/package.json:30]: And here I thought I was being smart.  Ill revert the bumping stuff

File: yarn.lock
Comment (mortennordseth) [inline yarn.lock:1]: Looks like the yarn.lock changed from yarn registry to npm registry. I guess this was not intended and that these changes should be reverted 😃

Comment (tft-abel) [reply yarn.lock:1]: Fixed (/)

**Review by mortennordseth — APPROVED** [review]
🚀

**Pull request merged**